README.md

```diff
@@ -47,6 +47,54 @@ But I will be accepting PRs to this repository.
 
 I use eclipse neon for development. You can simply import Minebot as existing project.
 
+================== Message from 1.16.3 Porter (Vaccinate) =============
```

Comment:
This is a markdown file.

You can start headlines by using:
```
## My Headline
```
Single linebreaks are ignored.